Runtime/Game/LootLockerGameServerAPI.cs

```diff
@@ -72,17 +72,17 @@ protected override void RefreshTokenAndCompleteCall(LootLockerServerRequest cach
                 });
             }
 
-            void CompleteCall(LootLockerServerRequest cacheServerRequest, Action<LootLockerResponse> OnServerResponse, LootLockerSessionResponse response)
+            void CompleteCall(LootLockerServerRequest newcacheServerRequest, Action<LootLockerResponse> newOnServerResponse, LootLockerSessionResponse response)
```

Comment:
This is not supported, using the same variable name and overriding it in a function inside of a function, that's why this needed to be changed. So this part should not create any bugs.